eman2/convert/convert.py

```diff
@@ -414,3 +511,137 @@ def calculatePhaseShift(ampcont):
     ctfPhaseShift = numpy.rad2deg(PhaseShift)
 
     return ctfPhaseShift
+
+
+def coordinates2json(pathInputCoor, inputCoor):
```

Comment:
I'd refer in the name that is a 3dcoordinate method: coordinate3d2json?

Reply:
Actually, this can simply go away as it is not used at the moment (some old code). Fixed here https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/3ab25a6fd8a7029dfe0412c7b336e06d62560ed7